ysoserial/Generators/ActivitySurrogateSelectorFromFileGenerator.cs

```diff
@@ -16,6 +16,10 @@ protected PayloadClassFromFile(SerializationInfo info, StreamingContext context)
 
         public PayloadClassFromFile(string file)
         {
+            if(file.StartsWith("cmd /c "))
```

Comment:
what for?

Reply:
As I have removed "cmd /c " from all the payloads to add support for --rawcmd, this will be added to the user command when the --rawcmd option hasn't been provided. So all commands without --rawcmd will start with "cmd /c " now. --rawcmd can help making the payload shorter or to use something like powershell instead of cmd.